metal/datasource_metal_connection.go

```diff
@@ -162,7 +161,7 @@ func dataSourceMetalConnectionRead(d *schema.ResourceData, meta interface{}) err
 		d.Set("facility", conn.Facility.Code)
 	}
 	if conn.Metro != nil {
-		d.Set("metro", conn.Metro.Code)
+		d.Set("metro", toLower(conn.Metro.Code))
```

Comment:
perhaps we should just use `strings.ToLower` and avoid the `interface{}` wrapping.